donation/serializers.py

```diff
@@ -61,3 +33,49 @@ def create(self, validated_data):
             search_slug=self.switch(validated_data['blood_group'])
         )
         return donation_request
+
+    def update(self, instance, validated_data):
+        instance.blood_group = validated_data.get(
```

Comment:
why are we manually attaching related objects?

Reply:
I didn't see any other approach except manually attaching logged in user to the request through serializers.........